doc/devdoc/cppcomments.rst

```diff
@@ -0,0 +1,40 @@
+Rules for C++  code comments
+============================
+
```

Comment:
Maybe add a link to doxygen documentation?

Reply:
done